test/chrome_test.dart

```diff
@@ -28,18 +28,17 @@ Future<Uri> sourceUriProvider(String sourceUrl, String scriptId) async =>
 
 void main() {
   test('reports correctly', () async {
-    final preciseCoverage = json.decode(
+    final List preciseCoverage = json.decode(
```

Comment:
This is not how we want to handle casts.

```dart
final preciseCoverage = json.decode(...) as List;
```

I'd prefer to fix the package analysis options than to expand use of implicit casts

Reply:
Agreed. Let's do that in a follow-up. There are a lot of things to clean up.